ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+bash ci/codecov.sh
```

Comment:
Oh, we probably need to export some `TRAVIS_` variables so codecov can detect which CI we are using, but we can probably tell it that too via command-line options: https://github.com/codecov/codecov-bash/blob/master/codecov#L78-L176